tutorials/tutorial2.md

```diff
@@ -0,0 +1,140 @@
+
+If you want to train your own AtacWorks model instead of using the model reported in the paper, refer to Tutorial 1 (link).
+
+
+## Step 1: Set parameters
+
+```
+atacworks=<path to atacworks>
+```
+
+## Step 2: Download model
+
+Download a pre-trained deep learning model (model.pth.tar) trained with dsc-ATAC-seq data from Monocytes and B cells. This model was reported and used in the AtacWorks paper (1).
+```
+aws s3 cp s3://atacworks-paper/dsc_atac_blood_cell_denoising_experiments/models/model.pth.tar ./models/model.pth.tar
+```
+
+## Step 3: Download config files
+
+We also need to download the 'configs' directory containing config files for this experiment. The config files describe the parameters of the experiment, including the structure of the deep learning model.
+```
+aws s3 cp s3://atacworks-paper/dsc_atac_blood_cell_denoising_experiments/configs/ ./configs --recursive
+```
+
+## Step 4: Download the test dsc-ATAC-seq signal from 50 NK cells (~ 1M reads), in bigWig format
+
+```
+aws s3 cp s3://atacworks-paper/dsc_atac_blood_cell_denoising_experiments/test_data/noisy_data/dsc.1.NK.50.cutsites.smoothed.200.bw ./
+```
+
+## Step 5: Download genomic intervals for testing
+
+The model we downloaded takes the input ATAC-seq signal in genomic intervals spanning 50,000 bp. The genomic positions of the intervals to use are supplied to the model in BED format. In this example, we test the model on intervals that span all autosomes except for chromosome 10 and chromosome 20. See get_intervals.py (link) for help in creating intervals spanning the whole genome or a different set of chromosomes.
+```
+aws s3 cp s3://atacworks-paper/dsc_atac_blood_cell_denoising_experiments/intervals/hg19.50000.genome_intervals.bed ./intervals/hg19.50000.genome_intervals.bed
+```
+
+## Step 6: Read test data over the selected intervals, and save in .h5 format
+
+We supply to `bw2h5.py` the bigWig file containing the noisy ATAC-seq signal, and the BED file containing the intervals on which to apply the model. This script reads the ATAC-seq signal within each supplied interval and saves it to a .h5 file.
+
+```
+python $atacworks/bw2h5.py \
+           --noisybw dsc.1.NK.50.cutsites.smoothed.200.bw \
+           --intervals intervals/hg19.50000.genome_intervals.bed \
+           --prefix NK.50_cells \
+           --pad 5000 \
+           --batch_size 2000 \
+           --nolabel
+```
+This creates a file `NK.50_cells.h5`, which contains the noisy ATAC-seq signal to be fed to the pre-trained model.
+
+## Step 7: Inference on selected intervals, producing denoised track and binary peak calls
+
+```
+python $atacworks/main.py --infer \
```

Comment:
In this command, replace `--sizes` with `--sizes_file`. there is no `--sizes`. Additionally, `--field` is missing. Provide it either in the config file `field: 1000` or as a command line argument `--field 1000`.

Reply:
fixed the --sizes_file.

Reply:
this should be fixed now.